torch/fx/experimental/symbolic_shapes.py

```diff
@@ -195,6 +196,24 @@ def guard_scalar(a):
 
 # inclusive both ways
 def constrain_range(a, *, min: Optional[int], max: Optional[int] = None):
+    """
+    Constrain range takes a symbol, and records a valid value range for it in the graph.
+    That value range is the narrowest intersection of the current value range for the symbol.
```

Comment:
nit: it's weird to say "narrowest" here because the intersection of current and new value range is unique